app/Http/Controllers/AdminsController.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\User;
+use Hash;
+use Validator;
+
+class AdminsController extends Controller
+{
+
+    function NewAdmin (Request $r)
+    {
+    	$validation=Validator::make($r->all(),
+    	[
+    		"name"=>"required",
+    		"surname"=>"required",
+    		"email"=>"required|email|unique:users,email",
+    		"phone"=>"nullable|numeric",
+    		"password"=>"required|min:6",
+    		"config_password"=>"required|same:password",
+    		"is_active"=>"required|integer",
+    	]);
+
+    	if ($validation->fails()) 
```

Comment:
Don't use newline between "if" and "{".
if (condition == something) {
  // your code here
}